examples/with-mqtt-js/README.md

```diff
@@ -0,0 +1,15 @@
+# MQTT Client example
+
+This example shows how to use the MQTT.js client with next.js.
+Thanks @jamesmosier for the startup advice.
+
+## How to use
+
+npm install or npm i
+
+Define a connection URI, options and handlers per topic for incomming messages.
+Then pass them to the useMqtt hook that returns a connected client instance.
+
+## MQTT.js
+
+For more information on the mqtt client or the options object, consult the documentation: [MQTT.js](https://github.com/mqttjs/MQTT.js)
```

Comment:
The readme should follow the same structure of our other examples. For example: https://github.com/vercel/next.js/tree/canary/examples/hello-world - If there are requirements for setting up environment variables, you need to be clear on those (eg: https://github.com/vercel/next.js/tree/canary/examples/cms-buttercms)